### PR TITLE
Drop some aliases

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

**Comment by traxys** [comment]
`none-ls` seemed to rely on the entry being named `treefmt`, you need to set `treefmt = treemft2` instead of the inherit

We have some buildbot issues that drown real test failures, sorry :/

**Comment by MattSturgeon** [comment]
Thanks for this!

In addition to @traxys feedback (`treefmt = treemft2`), could you reword the commit using the format `plugins/{lsp,none-ls}: remove package aliases`?

Or if you prefer two commits:
- `plugins/lsp: remove typescript-language-server alias`
- `plugins/none-ls: remove treefmt + markdownlint-cli aliases`

**Comment by SuperSandro2000** [comment]
both things done :)

**Review by GaetanLepage — APPROVED** [review]
Thanks @SuperSandro2000 !

**Comment by GaetanLepage** [comment]
https://buildbot.nix-community.org/#/builders/43/builds/266/steps/2/logs/stdio

**Comment by MattSturgeon** [comment]
Thanks!

Until #1878 is resolved, we'll have to test locally.

File: plugins/none-ls/packages.nix [hunk 1]
```diff
@@ -173,6 +172,7 @@ pkgs: {
       terraform_fmt = pkgs.terraform;
       terraform_validate = pkgs.terraform;
       tidy = pkgs.html-tidy;
+      treefmt = pkgs.treemft2;
```
Comment (SuperSandro2000) [inline plugins/none-ls/packages.nix:175]: bruh...

**Comment by SuperSandro2000** [comment]
I normally rebuild my config with the changes before opening the PR but I regularly forget it when applying review changes 🤦🏼

**Review by MattSturgeon — APPROVED** [review]
LGTM, should be simple enough we can merge without CI

**Pull request merged**